ibm_catalog.json

```diff
@@ -0,0 +1,574 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vsi-vpc",
+      "label": "[Add-ons Beta] Cloud automation for Virtual Server Instances for VPC",
```

Comment:
According to the [docs](https://cloud.ibm.com/docs/vpc?topic=vpc-about-advanced-virtual-servers), the official service name is "IBM Cloud® Virtual Servers for Virtual Private Cloud". We should probably be consistent with this? Perhaps `[Add-ons Beta] Cloud automation for Virtual Servers for Virtual Private Cloud` ?